build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
       dagger              : '2.16',
       butterKnife         : '8.8.1',
       kotterKnife         : 'e157638df1',
-      room                : '1.1.0',
+      room                : '1.1.1',
```

Comment:
There seems to be a [bug in Room 1.1.0](https://stackoverflow.com/questions/50420649/upgrading-room-db-with-migrations-causes-db-lock) where the database gets locked once migrations are running. Based on the SO comments in the link, upgrading to `1.1.1` fixes the issue.